Node/core/src/bots/ChatConnector.ts

```diff
@@ -150,6 +152,17 @@ export class ChatConnector implements IConnector, IBotStorage {
         };
     }
 
+    private getRefreshEndpoint(channelAuthTenant?: string): string {
+        var tenant = channelAuthTenant && channelAuthTenant.length > 0
+            ? channelAuthTenant
+            : "botframework.com";
+
+        var endpoint = 'https://login.microsoftonline.com/' + tenant + '/oauth2/v2.0/token';
```

Comment:
constants should go in better locations.

Reply:
yep